src/app/components/route-block/index.js

```diff
@@ -33,196 +38,36 @@ export function RouteBlock() {
-										{ ' ' }
-										{ blockType.description }
-									</div>
-									<div className="bb-route-block__row">
-										<strong>Category:</strong>
-										{ ' ' }
-										{ blockType.category }
-									</div>
-									{ !! blockType?.keywords.length && (
-										<div className="bb-route-block__row">
-											<strong>Keywords:</strong>
-											{ ' ' }
-											{ blockType.keywords.join( ', ' ) }
-										</div>
-									) }
-									{ !! Object.keys( blockType.supports ?? {} )
-										.length && (
-										<div className="bb-route-block__supports bb-route-block__row">
-											<div>
-												<h2>Supports</h2>
-											</div>
-											<ul className="bb-route-block__supports--list">
-												{ Object.entries(
-													blockType.supports
-												).map( ( [ key, support ] ) => (
-													<li key={ key }>
-														<strong>{ key }</strong>{ ' ' }
-														{ `- ${ JSON.stringify(
-															support
-														) }` }
-													</li>
-												) ) }
-											</ul>
-										</div>
-									) }
-
-									{ !! Object.keys( blockType.styles ?? {} )
-										.length && (
-										<div>
-											<div className="bb-route-block__styles bb-route-block__row">
-												<h2>Styles</h2>
-											</div>
-											<ul className="bb-route-block__styles--list">
-												{ Object.entries(
-													blockType.styles
-												).map( ( [ key, style ] ) => (
-													<li key={ key }>
-														<strong>
-															{ style.label }
-														</strong>
-														{ ! style.isDefault && (
-															<>
-																{ ': ' }
-																<code>
-																	{ `is-style-${ style.name }` }{ ' ' }
-																</code>
-															</>
-														) }
-													</li>
-												) ) }
-											</ul>
-										</div>
-									) }
-								</div>
-								<div>
-									<div>
-										<h2>Attributes</h2>
-									</div>
-									<ul className="bb-route-block__attributes">
-										{ Object.entries(
-											blockType.attributes
-										).map( ( [ key, attributeScheme ] ) => (
-											<li key={ key }>
-												<strong>{ key }</strong>{ ' ' }
-												{ attributeScheme.type &&
-													`(${ attributeScheme.type })` }
-											</li>
-										) ) }
-									</ul>
-								</div>
-							</div>
-						);
-					}
-
-					if ( currentPanel.name === 'example' ) {
-						return (
-							<div className="bb-route-block__example">
-								<BlockPreview
-									viewportWidth={ 1000 }
-									blocks={ getBlockFromExample(
-										blockType.name,
-										{
-											attributes:
-												blockType.example.attributes,
-											innerBlocks:
-												blockType.example.innerBlocks,
-										}
-									) }
-								/>
-							</div>
-						);
-					}
+			<ButtonGroup>
+				{ tabs.map( ( tab, key ) => {
+					const destination =
```

Comment:
this could have been a key of the `tabs` array above to avoid the special casing.

Reply:
I changed it so that the `tab.name` is used as the key